lib/react-markdown.js

```diff
@@ -13,6 +13,7 @@
  * @property {PluggableList} [plugins=[]] **deprecated**: use `remarkPlugins` instead
  * @property {PluggableList} [remarkPlugins=[]]
  * @property {PluggableList} [rehypePlugins=[]]
+ * @property {import('remark-rehype').Options} [rehypeOptions={}]
```

Comment:
I don’t think this name works well. rehype itself doesn’t have options. `remarkRehype` has options.

What about `remarkRehypeOptions`?